src/client/getDataFromStack.js

```diff
@@ -1,29 +1,43 @@
 import URL from '../server/url.js';
 
-export async function getData () {
-    try {
-        const response = await fetch(URL);
-        const json = await response.json();
 
-        let tableHtml = '';
+function wrapStackObject(json){
+    let tableHtml = '';
+    for (const code in json) {
```

Comment:
Please separate code blocks and multiline statements with newlines here and in several places below.